db/models/articles.models.js

```diff
@@ -69,3 +59,17 @@ exports.selectCommentsFromArticleId = (id) => {
       return rows;
     });
 };
+
+exports.insertComments = (articleId, {body, username}) => {
+    if (!body.length){
```

Comment:
We've hardcode this error here meaning it only works if the body is empty. Could we also consider what will happen if the username is blank or doesn't exist on the input body? Will psql throw an error for us or do we need to handle it with a custom error?